src/dispatch/incident/flows.py

```diff
@@ -493,6 +493,18 @@ def incident_active_flow(incident_id: int, command: Optional[dict] = None, db_se
     # we load the incident instance
     incident = incident_service.get(db_session=db_session, incident_id=incident_id)
 
+    # we get the incident document
+    incident_document = get_document(
+        db_session=db_session,
+        incident_id=incident_id,
+        resource_type=INCIDENT_RESOURCE_INVESTIGATION_DOCUMENT,
+    )
+
+    # update the incident document
+    document_updates = {f"Status: {incident.status}": f"Status: Active"}
+    document_plugin = plugins.get(INCIDENT_PLUGIN_DOCUMENT_SLUG)
+    document_plugin.update(incident_document.id, document_updates, raw=True)
```

Comment:
@kevgliss I'd move it after the status check and before updating the incident status, so we only update the doc when the status of the incident is not active.

Reply:
Same for the other flows.